app/assets/javascripts/discourse/app/lib/ajax-error.js

```diff
@@ -69,6 +70,11 @@ export function popupAjaxError(error) {
   }
   bootbox.alert(extractError(error));
 
+  // in testing mode we want to be able to test these ajax popup messages
+  if (isTesting()) {
```

Comment:
I am not sure if this will come back to bite me...I think if all the acceptance tests pass it should be OK

Reply:
Well they ran okay! As a little more context I wanted to test a bad response here:

https://github.com/discourse/discourse/pull/13083/files#diff-785a91ce827d2667a7e90342377d3f89c3e8c6715f8f75a5efc9894f41fb479dR201-R206

But if I just did that the actual error was thrown and stopped the tests, even though the popup shows the message :/